### ISSUE TITLE
File Upload In Demo Not Working On IOS - No MultiFormat Readers Were Able To Detect The Code

### ISSUE THREAD

**Issue by wickedvic** [mebjas/html5-qrcode#692]
**To Reproduce**
1. Go to the demo at [https://blog.minhazav.dev/research/html5-qrcode](url)
2. Open same URL on web and on IPhone
3. Using image below, web and android are able to read the code 128 barcode. IOS throws an error of "No MultiFormat Readers Were Able To Detect The Code"
4. Screenshots posted below. Image 1 is the barcode used. Image 2 is view in chrome on MacOS. Image 3 is on IOS IPhone 13

**Expected behavior**
1. Decode same barcode on IOS as well

<img width="1003" alt="Code128" src="https://user-images.githubusercontent.com/70414907/217445923-8dcce439-8329-4c48-bb9e-42dbd4bbb8fa.png">

<img width="957" alt="Screen Shot 2023-02-08 at 12 53 06 AM" src="https://user-images.githubusercontent.com/70414907/217446288-8ac67687-8139-4431-9def-e02fd6ac6835.png">

![IMG_1252](https://user-images.githubusercontent.com/70414907/217446648-ed52de13-c261-412d-bb29-0fb193a532b1.png)

**Comment by ROBERT-MCDOWELL** [comment]
your barcode is in negative color which is not (yet) supported.

**Comment by wickedvic** [comment]
@ROBERT-MCDOWELL any reason this barcode also reproduces the same error? 

![Tube 6](https://user-images.githubusercontent.com/70414907/217711140-f7b0e9a9-9c7f-473f-a375-e4cee9e0099f.jpg)

**Comment by JR-SaversNest** [comment]
Hello,
First of all, thanks for the provided library! I've been trying to use it and the results are not fully successfull. For some barcodes, it works (few) and for other not so much.
I'm attaching a few pictures with barcodes I took. I've submitted to the scanapp but even there the codes are not recognized.
![IMG_3771](https://user-images.githubusercontent.com/123068608/220171408-95c38a26-5215-4239-92a0-9d71d4714821.jpg)
![IMG_3774](https://user-images.githubusercontent.com/123068608/220171412-94875783-c6b4-4e11-bd79-99900a26898f.jpg)
![IMG_3773](https://user-images.githubusercontent.com/123068608/220171416-e6eb8384-cf9e-47df-a75f-d08a5e3af3ff.jpg)

It only works when I zoom it via computer and scan it with my smarphone (iPhone 13  IOS 16) poiting to the zoomedin barcode.

Is there any adjustment possible to allow the successfull barcode recognition?

Many thanks!
João Ramos
